adoc/sql-server-getting-started-docinfo.xml

```diff
@@ -0,0 +1,22 @@
+<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+    <dm:bugtracker>
+        <dm:url>https://github.com/SUSE/suse-best-practices/issues/new</dm:url>
+        <dm:product>SUSE Enterprise Linux Server</dm:product>
```

Comment:
The product name here should be the title of the paper ;-), in this case
SQL Server on SUSE Linux Enterprise: Getting Started